onnxruntime/core/graph/graph.cc

```diff
@@ -876,9 +876,10 @@ void Node::CreateSubgraph(const std::string& attr_name) {
 
 void Node::AddAttributeProto(AttributeProto value) {
   utils::SetNodeAttribute(std::move(value), attributes_);
-
-  graph_->SetGraphResolveNeeded();
-  graph_->SetGraphProtoSyncNeeded();
+  if (graph_) {
```

Comment:
Is this change required due to the changes in this PR ?

Reply:
Absolutely, otherwise we have segment fault on unit test.